docs/.vuepress/containers/helpers/index.js

```diff
@@ -0,0 +1,46 @@
+const {
+  getEnvDocsFramework,
+  parseFramework
+} = require('../../helpers');
+
+/**
+ * Given the frontmatter object available in the Vuepress containers, returns the number of lines the frontmatter is
+ * taking in the original `.md` file.
+ *
+ * @param {object} frontMatterObj Frontmatter information object passed to the Vuepress' `env` argument.
+ * @returns {number} The number of lines taken by the frontmatter.
+ */
+function getContainerFrontMatterLength(frontMatterObj) {
+  const frontMatterEntriesCount = Object.keys(frontMatterObj).reduce(
+    (sum, key) => {
+      if (Array.isArray(frontMatterObj[key])) {
+        return sum + frontMatterObj[key].length + 1;
+      }
+
+      return sum + 1;
+    }, 0);
```

Comment:
The same suggestion [as here](https://github.com/handsontable/handsontable/pull/9319#discussion_r859761925).